nautobot_ssot/jobs/examples.py

```diff
@@ -94,14 +126,100 @@ class TenantModel(NautobotModel):
     _model = Tenant
     _modelname = "tenant"
     _identifiers = ("name",)
-    _children = {"prefix": "prefixes"}
+    _children = {}
 
     name: str
     prefixes: List[PrefixModel] = []
 
     pk: Optional[UUID]
 
 
+class DeviceTypeModel(NautobotModel):
+    """Shared data model representing a DeviceType in either of the local or remote Nautobot instances."""
+
+    _model = DeviceType
+    _modelname = "device_type"
+    _identifiers = ("model", "manufacturer__name")
+    _attributes = ("part_number", "u_height", "is_full_depth")
+
+    model: str
+    manufacturer__name: str
+    part_number: str
+    u_height: int
+    is_full_depth: bool
+
+    # Not in _attributes or _identifiers, hence not included in diff calculations
+    pk: Optional[UUID]
```

Comment:
With the newest iteration of contrib, you get these out of the box, no need to explicitly annotate them - pk should be available on all `NautobotModel`s

Reply:
That's what I thought but it appeared to be used when communicating to the remote Nautobot instance.

Reply:
Ahhh, yeah got it